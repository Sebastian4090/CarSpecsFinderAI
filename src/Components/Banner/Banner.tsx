const Banner = () => {
  return (
    <div className="w-full flex justify-center items-center bg-indigo-800 h-48 shadow-2xl select-none">
      <div className="w-9/12 sm:w-8/12 lg:w-1/2 h-24 bg-indigo-950 rounded-lg shadow-lg flex items-center">
        <h1 className="flex-auto text-2xl sm:text-4xl lg:text-4xl font-primary text-slate-200 text-center cursor-pointer">
          Car Specifications Finder
        </h1>
      </div>
    </div>
  );
};

export default Banner;
