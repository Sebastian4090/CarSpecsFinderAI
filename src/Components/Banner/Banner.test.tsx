import { expect, it } from "vitest";
import Banner from "./Banner";

it("renders correctly", () => {
  expect(<Banner />).toMatchSnapshot();
});
