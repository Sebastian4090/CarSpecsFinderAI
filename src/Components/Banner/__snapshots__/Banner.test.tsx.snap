// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<Banner /> Component > renders correctly 1`] = `<Banner />`;

exports[`renders correctly 1`] = `<Banner />`;
